README.md

```diff
@@ -29,9 +29,15 @@ Version 2 of the Envoy API will evolve existing APIs and introduce new APIs to:
 
 ## Status
 
-Draft work-in-progress. Input is welcome via issue filing. Small, localized PRs
-are also welcome, but any major changes or suggestions should be coordinated in
-a tracking issue with the authors.
+The LDS/CDS/EDS/RDS APIs are now frozen and will maintain backwards
+compatibility according to standard proto rules (e.g. new fields will not reuse
+tags, field types will not change, fields will not be renumbered, etc.).
```

Comment:
I had a quick look around, didn't find any official resource on this. I think the example provided give the idea, you can find some explicit discussion of backwards compatibility issues for specific proto types at placed such as https://developers.google.com/protocol-buffers/docs/proto3#backwards-compatibility-issues.